packages/contracts-core/contracts/test/fork/ForkBase.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import "forge-std/Test.sol";
+import {UpgradeCallBatchLogic} from "@nomad-xyz/contracts-ops/contracts/scripts/UpgradeCallBatches.s.sol";
+import {Home} from "../harnesses/HomeHarness.sol";
+import {ReplicaHarness} from "../harnesses/ReplicaHarness.sol";
+import {GovernanceRouterHarness} from "../harnesses/GovernanceRouterHarness.sol";
+import {XAppConnectionManager} from "../../XAppConnectionManager.sol";
+import {UpdaterManager} from "../../UpdaterManager.sol";
+
+contract ForkBase is UpgradeCallBatchLogic, Test {
+    Home home;
+    ReplicaHarness replica;
+    GovernanceRouterHarness governanceRouter;
+    XAppConnectionManager xAppConnectionManager;
+    UpdaterManager updaterManager;
+
+    string configName;
+    string govDomain;
+
+    string local;
+    string remote;
+    uint32 remoteDomain;
+    // Defined in CallBatch, which is inherited via UpgradeCallBatchLogic
+    // uint32 localDomain;
```

Comment:
this has created conflicts with bridge router tests as well. we should consider renaming callbatch's to `batchLocalDomain` ?

Reply:
LGTM